### PR TITLE
Upgrade/11.0 base technical features

### PR DESCRIPTION
**Pull request by StefanRijnhart**

### CODE REVIEW

File: base_technical_features/models/ir_ui_menu.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# coding: utf-8
+# © 2016 Opener B.V. (<https://opener.am>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, models
```
Comment (bealdav) [inline base_technical_features/models/ir_ui_menu.py]: openerp ? opener ! ? :D

**Review by bealdav — COMMENTED** [review]
Typo I suppose

**Review by legalsylvain — APPROVED** [review]
the two last commits 10b5c9d and 251a34b seems to be quiet unrelated to the purpose of the PR (porting base_technical_feature module), but well, as it is the first module to be ported in V11, not a big deal.
Thanks a lot. :+1:  (code review, no test.)

File: base_technical_features/README.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+   :target: https://runbot.odoo-community.org/runbot/149/10.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/server-tools/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed `feedback
+<https://github.com/OCA/
```
Comment (elicoidal) [inline base_technical_features/README.rst]: Remove this last part

File: base_technical_features/README.rst [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+In the background, this preference is mapped to the *Technical feature (w/o
+debug mode)* group that this module adds. As an administrator, you can
+therefore manage this preference from the regular Users and Groups menu items.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/149/10.0
```
Comment (elicoidal) [inline base_technical_features/README.rst]: 11

**Review by elicoidal — APPROVED** [review]
some details. LGTM

**Comment by StefanRijnhart** [comment]
Thanks for the reviews! @bealdav @elicoidal please review the fixes.

**Review by bealdav — APPROVED** [review]

**Review by atchuthan — APPROVED** [review]

**Review by moylop260 — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please move this PR to https://github.com/OCA/server-ux according #1006

**Comment by StefanRijnhart** [comment]
https://github.com/OCA/server-ux/pull/2

**Pull request closed without merging**